components/picker/stories/picker.stories.js

```diff
@@ -52,6 +52,16 @@ export default {
 		},
 		isQuiet,
 		isOpen,
+		menuHeight: {
+			name: "Max Menu Height",
```

Comment:
I would recommend renaming it to "Menu max block size" (doesn't sound as nice, but I would be for accuracy) - what do you think?